pkg/resourceinterpreter/configurableinterpreter/luavm/kube.go

```diff
@@ -37,6 +37,7 @@ func KubeLoader(ls *lua.LState) int {
 var kubeFuncs = map[string]lua.LGFunction{
 	"resourceAdd":             resourceAdd,
 	"accuratePodRequirements": accuratePodRequirements,
+	"accuratePodDependencies": accuratePodDependencies,
```

Comment:
I don't understand why it's named accurateXXX,  which seems to be more appropriate to use getXXX.

Reply:
It sounds good! It's named `accurateXXX` just for tidy. `getXXX` is more appropriate, I will fix it soon.